src/TrackGenerator.cpp

```diff
@@ -261,21 +282,34 @@ FP_PRECISION* TrackGenerator::getFSRVolumes() {
   FP_PRECISION* FSR_volumes = new FP_PRECISION[num_FSRs];
   memset(FSR_volumes, 0., num_FSRs*sizeof(FP_PRECISION));
 
-  int azim_index;
-  segment* curr_segment;
-  FP_PRECISION volume;
-
   /* Calculate each FSR's "volume" by accumulating the total length of *
    * all Track segments multipled by the Track "widths" for each FSR.  */
-  for (int i=0; i < _num_azim; i++) {
-    for (int j=0; j < _num_tracks[i]; j++) {
+#pragma omp parallel
+  {
 
-      azim_index = _tracks[i][j].getAzimAngleIndex();
+    int azim_index, fsr_id;
+    segment* curr_segment;
+    FP_PRECISION volume;
 
-      for (int s=0; s < _tracks[i][j].getNumSegments(); s++) {
-        curr_segment = _tracks[i][j].getSegment(s);
-        volume = curr_segment->_length * _azim_weights[azim_index];
-        FSR_volumes[curr_segment->_region_id] += volume;
+    for (int i=0; i < _num_azim; i++) {
+#pragma omp for
```

Comment:
According to the google c++ style guide, preprocessor directives should not be indented. I sneaked in a PR a week ago removing the indents on the pragmas so have kept that consistency here. I also don't have a strong preference either way, but lean towards removing them because the google c++ style guide emacs plugin likes to remove the indents when it auto-indents. That said, they do stand out quite a bit.